pov_tools/create_POV_data_files

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
+# create_POV_data_files
+# version - all .png files that don't contain 'preview' are processed
+OIFS="$IFS"
+IFS=$'\n'
+cd "$(dirname "$0")"
+clear
+height_only=0
+preset=0
+download=0
+
+echo ""
+echo "Welcome! Let's make some POV images!"
+echo ""
+echo "Found these image files:"
+# get .png files except containing 'preview' and build array
+pngs=($(find . -maxdepth 1 -type f -name "*.png" ! -name "*preview*"))
+for strip in ${pngs[@]}; do echo "${strip##*/}"; done
+# get number of files in array
+files=${#pngs[@]}
+echo ""
+
+# If NO images in the folder, offer logo
+if [[ $files -lt 1 ]]; then
+	echo ""
+	echo "OOPS! No .png image files found!"
+	echo ""
+	echo "Do you want to download a sample Star Wars logo image?"
+	echo "Type y for yes,"
+	echo "type q to quit,"
+	read -p $'or press ENTER to start over..\n' dl
+	case $dl in
+		y) download=1;;
+		q) exit;;
+		*) exec "./create_POV_data_files";;
+	esac
+fi
+if [[ $download == 1 ]]; then
+	echo ""
+	wget https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Star_Wars_Logo.svg/1024px-Star_Wars_Logo.svg.png
+	pngs=$"1024px-Star_Wars_Logo.svg.png"
+fi
+
+valid_option=0
+while [[ $valid_option != 1 ]]; do
+	echo "Please select the settings you want to create the POV image(s) with:"
+	echo "	height - number of pixels in your blade (one strip emitter to tip)"
+	echo "	length - blade length in the measurement unit of choice"
+	echo "	offset - the distance (in same unit as L) from the bottom of the blade"
```

Comment:
"(in same unit as L)" should be: "(in same unit as length)". This is probably still from when there was L for length in the comments)

Reply:
fixed